pkg/util/defaults/defaults.go

```diff
@@ -26,7 +26,7 @@ const (
 	Version = "2.0.0-SNAPSHOT"
 
 	// DefaultRuntimeVersion --
-	DefaultRuntimeVersion = "1.18.0-SNAPSHOT"
+	DefaultRuntimeVersion = "3.20.1-SNAPSHOT"
```

Comment:
This was changed automatically, should I revert this change ?

Reply:
No, it's okey to keep it.